packages/plugin-ext-vscode/package.json

```diff
@@ -10,6 +10,7 @@
     "@theia/plugin-ext": "^0.16.0",
     "@theia/workspace": "^0.16.0",
     "@types/request": "^2.0.3",
+    "filenamify": "^4.1.0",
```

Comment:
If not already done so, we will need to verify using the [process](https://github.com/eclipse-theia/theia/wiki/Registering-CQs#case-new-or-updated-npm-production-dependency) if the new dependency is license-certified.

Reply:
it is MIT as well as its dependencies